crates/core/src/delta_datafusion/mod.rs

```diff
@@ -3059,8 +3059,8 @@ mod tests {
         assert_eq!("a", small.iter().next().unwrap().unwrap());
 
         let expected = vec![
-            ObjectStoreOperation::GetRange(LocationType::Data, 4920..4928),
-            ObjectStoreOperation::GetRange(LocationType::Data, 2399..4920),
+            ObjectStoreOperation::GetRange(LocationType::Data, 4952..4960),
```

Comment:
Parquet files get slightly larger in some cases, I think due to 
- https://github.com/apache/arrow-rs/pull/7643